### ISSUE TITLE
Refused to load manifest from 'http://localhost:3000/app/assets/img/site.webmanifest'

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#9745]
Seeing this in browser console on 0.32.3 when running locally. 

```
Refused to load manifest from 'http://localhost:3000/app/assets/img/site.webmanifest' because it violates the following Content Security Policy directive: "default-src 'none'". Note that 'manifest-src' was not explicitly set, so 'default-src' is used as a fallback.
```

Looks like a new `site.webmanifest` file was added in #9211 with these contents:

```json
{
    "name": "Metabase",
    "short_name": "Metabase",
    "icons": [
        {
            "src": "/app/assets/img/android-chrome-192x192.png",
            "sizes": "192x192",
            "type": "image/png"
        },
        {
            "src": "/app/assets/img/android-chrome-256x256.png",
            "sizes": "256x256",
            "type": "image/png"
        }
    ],
    "theme_color": "#ffffff",
    "background_color": "#ffffff",
    "display": "standalone"
}
```

Not sure this file is needed because browsers that follow our CSP directives have been ignoring the file all along. If it was needed someone probably would have noticed it wasn't working when working on the fix in that PR 🤔 

We can remove the file if it's indeed not needed or whitelist the file if we do need it

**Comment by camsaul** [comment]
Ping @tlrobinson @kdoh. Do we want this file or not

**Comment by DieterMartin** [comment]
I am having a similar message when trying to create a pulse, i get the same above error and when i subsequently try to load a question to the pulse it just hangs indefinitely.  is there a fix for this as none of my pulses are working at this point.

**Comment by camsaul** [comment]
@DieterMartin this message is a warning and will not affect Pulse creation, it is unrelated

**Comment by camsaul** [comment]
@tlrobinson @kdoh I fixed this since I never heard back. Assumed it's something we wanted.

**Comment by camsaul** [comment]
Fixed by #9899
